scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -1006,7 +1008,7 @@ def change_skip_to_error_if_integration(options, instance):
         # Do not treat this as error if filter type is command line
         filters = {t['type'] for t in instance.filters}
         ignore_filters ={Filters.CMD_LINE, Filters.SKIP, Filters.PLATFORM_KEY,
-                         Filters.TOOLCHAIN, Filters.MODULE}
+                         Filters.TOOLCHAIN, Filters.MODULE, Filters.TESTPLAN}
```

Comment:
if i understand the logic here, it looks like a list of what should be considered as a 'error skip' on integration will be shorter with only `TESTSUITE` and `PLATFORM` (considering `QUARENTINE` is already taken into account with if condition on `instance.reason`)

Reply:
My reasoning behind why I think the current way is safer: https://github.com/zephyrproject-rtos/zephyr/pull/62810#discussion_r1423923101